project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss

```diff
@@ -7,6 +7,15 @@
   background: #f8f8f8 !important;
 }
 
+.image {
+  position: absolute !important;
+  margin-top: 2px;
+  height: 20px !important;
+}
+.margin-left-xxs
```

Comment:
use css beautifier plugin in your code editor for the auto alignment of the classes in css files

Reply:
Added the plugin Css beautifier.